spec/cypress/integration/admin-create-administrator.js

```diff
@@ -0,0 +1,39 @@
+describe("Admin user creating another admin user", () => {
+  beforeEach(() => {
+    cy.login("admin");
+  });
+
+  it("should show a create administrator button", () => {
+    cy.visit("/admin/administrators");
+    cy.get("a.govuk-button").contains("Create a new administrator");
```

Comment:
It's a tiny thing, but `.contain()` is designed for element selecting, not testing - it does test as well, but it would be better to use `.should('contain', 'Create a new…')` (or `eq` for an exact match).